pages/linux/slurp.md

```diff
@@ -0,0 +1,32 @@
+# slurp
+
+> Select a region in a Wayland compositor.
+> More information: <https://github.com/emersion/slurp>.
+
+- Select a region and print it to stdout:
+
+`slurp`
+
+- Show help message and quit:
```

Comment:
The help example should be the last example on the page, and we prefer "Display help" and "and quit" can be removed as this is what help commands usually do.

```suggestion
- Display help:
```